contracts/contracts/structs/CrossNet.sol

```diff
@@ -31,7 +31,7 @@ struct BottomUpCheckpoint {
     /// @dev Batch of messages to execute.
     IpcEnvelope[] msgs;
     /// @dev The activity summary from child subnet to parent subnet
-    ActivitySummary activities;
+    Consensus.Compressed activities;
```

Comment:
@raulk using compressed instead over full as only `Compressed` needs to be persisted with bottom up checkpoint in the parent, there is no need to commit Full report to the storage in the parent subnet.

Reply:
Absolutely. This was my IDE messing up.